distribution/src/main/packaging/systemd/elasticsearch@.service

```diff
@@ -0,0 +1,60 @@
+[Unit]
+Description=Elasticsearch Instance %i
+Documentation=http://www.elastic.co
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+RuntimeDirectory=elasticsearch
+Environment=ES_HOME=/usr/share/elasticsearch
+Environment=ES_PATH_CONF=${path.conf}/%i
+Environment=PID_DIR=/var/run/elasticsearch
+EnvironmentFile=-${path.env}-%i
```

Comment:
We don't _technically_ need the `EnvironmentFile` to be present, as the most important thing it's doing is setting `ES_PATH_CONF` which the unit already does. It may be a good idea to add an `AssertPathExists` for the `ES_PATH_CONF` directory, though?